resources/assets/coffee/react/profile-page/detail.coffee

```diff
@@ -32,7 +32,7 @@ export class Detail extends React.PureComponent
   constructor: (props) ->
     super props
 
-    @state = extended: true
+    @state = extended: props.user.ranking_extended
```

Comment:
if it's simple enough, rename `extended` to `expanded` to be inline with the naming for expanded elements elsewhere